sdk/appconfiguration/ci.yml

```diff
@@ -0,0 +1,33 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
@danieljurek What else do we need to do to setup the` cpp - appconfiguration - ci` steps?

Reply:
RUn the prepair pipelines pipeline - I sent you an email about it.

Reply:
Internal pipelines (when you create `cpp - appconfiguration`) need two variables set on them: 
* VcpkgForkBranchName
* VcpkgPRTitle

I think one or both of those are required because of https://github.com/Azure/azure-sdk-for-cpp/issues/4860. Let me know when the pipeline finishes and I can add these.

Reply:
> RUn the prepair pipelines pipeline - I sent you an email about it.

I ran it, using `refs/pull/6176/merge`

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4303171&view=results

Reply:
> Let me know when the pipeline finishes and I can add these.

The prepare pipelines step passed.